code/modules/mob/living/carbon/xenomorph/Xenomorph.dm

```diff
@@ -279,6 +279,8 @@
 	var/icon_xeno
 	var/icon_xenonid
 
+	var/shriek_status_flags = 0
```

Comment:
instead of adding a variable to every single xeno for this, make shriek enhancing a trait and check for that instead